### ISSUE TITLE
Portability issue

### ISSUE THREAD

**Issue by EU1KY** [NT7S/Si5351#2]
I'm successfully using your code for my antenna analyzer project (using the board based on STM32 ARM Cortex M3), but had to make some changes in si5351 code.
There are several platform-dependent definitions like TW_xxx used in si5351.c. It does not allow building the code for other platforms. All such checks should be moved to i2c.c which is platform dependent.

**Comment by NT7S** [comment]
Yes, as I only currently have Uno hardware, I admittedly did not write this first pass of code with portability in mind. I'll try moving the defs into i2c.c and see if that will do the trick.
